Plugins/ReleaseNotesGenerator/HtmlFragment.cs

```diff
@@ -157,27 +157,16 @@ internal static string To8DigitString(int x)
             return string.Format("{0:00000000}", x);
         }
 
-        /// <summary>
-        /// Clears clipboard and copy a HTML fragment to the clipboard. This generates the header.
-        /// </summary>
-        /// <param name="htmlFragment">A html fragment.</param>
-        /// <example>
-        ///    HtmlFragment.CopyToClipboard("<b>Hello!</b>");
-        /// </example>
-        public static void CopyToClipboard(string htmlFragment)
```

Comment:
I opted to remove this documentation as it didn't add anything over its sibling.